test/unit/sync/helpers/sync-helpers.test.ts

```diff
@@ -6,6 +6,8 @@ import {
 } from "../../../../src/sync/installation";
 import {getLogger} from "../../../../src/config/logger";
 
+jest.mock("../../../../src/models");
```

Comment:
I have no idea why I started getting this test failure after my changes:
```
FAIL test/unit/sync/installation.test.ts
  ● Test suite failed to run

    TypeError: Cannot read property 'init' of undefined

      28 | );
      29 |
    > 30 | SubscriptionModel.init(
         |                   ^
      31 | 	{
      32 | 		gitHubInstallationId: DataTypes.INTEGER,
      33 | 		jiraHost: DataTypes.STRING,

      at Object.<anonymous> (src/models/index.ts:30:19)
      at Object.<anonymous> (src/transforms/push.ts:1:1)
      at Object.<anonymous> (src/sqs/push.ts:3:1)
      at Object.<anonymous> (src/sqs/queues.ts:4:1)
      at Object.<anonymous> (src/models/subscription.ts:7:1)
 ```
 
 But adding this mock helped mitigating it